### PR TITLE
chore: Remove static api documentation

### PR DESCRIPTION
**Pull request by benjaminParisel**
* The api documentation is available on a external website

### CODE REVIEW

**Comment by benjaminParisel** [comment]
All redirect listed in the comments are good.

File: modules/ROOT/pages/parameters.adoc [hunk 1]
```diff
@@ -47,4 +47,4 @@ You can edit a parameters.properties file to change the value of a parameter bef
 
 == Parameter usage in forms
 
-From 7.x, to use parameters in forms, you need to retrieve them using the xref:bpm-api.adoc[REST API].
+From 7.x, to use parameters in forms, you need to retrieve them using the {openApiUrl}/latest#tag/BPM[REST API].
```
Comment (benjaminParisel) [inline modules/ROOT/pages/parameters.adoc]:
```suggestion
From 7.x, to use parameters in forms, you need to retrieve them using the {openApiUrl}/latest/#tag/ProcessParameter[REST API].
```

File: modules/ROOT/pages/rest-api-overview.adoc [hunk 2]
```diff
@@ -1,351 +1,39 @@
...
+=== View the API reference
 
-[#bonita-authentication]
-
-=== Authenticate to Bonita
-
-See xref:rest-api-authentication.adoc[REST Authentication] details to understand how to login from Bonita REST APIs.
-
-To log in, use the following request:
+You can view the reference for:
```
Comment (benjaminParisel) [inline modules/ROOT/pages/rest-api-overview.adoc]:
```suggestion
```

**Comment by benjaminParisel** [comment]
We tested with @uguy and we will merge it

**Review by uguy — APPROVED** [review]

**Pull request merged**